gemfiles/sass_3_4.gemfile

```diff
@@ -16,7 +18,6 @@ gem "pg"
 gem "rails", "4.2.2"
 gem "recipient_interceptor"
 gem "refills"
-gem "sass-rails", "~> 5.0"
```

Comment:
I was hoping someone could tell me. I ran the Appraisal installer and this was the result. I didn't edit it by hand.